src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/LibuvOutputConsumer.cs

```diff
@@ -67,18 +67,6 @@ public class LibuvOutputConsumer
 
                     if (result.IsCancelled)
                     {
-                        // Send a FIN
-                        _log.ConnectionWriteFin(_connectionId);
-
-                        using (var shutdownReq = new UvShutdownReq(_log))
```

Comment:
I think this is the only place where we use `UvShutdownReq`. Can we remove the type altogether? Or do you prefer to keep it around?